server/src/organizations/entities/organization.entity.ts

```diff
@@ -27,8 +27,21 @@ export class Organization {
   state: string;
 
   @Column({ type: 'int', unique: true })
+  // validator from api might go here
+  // why does it skip the id if it doesn't work
   ein: number;
 
   @Column('int')
   tax_exempt_id: number;
+
+  @AfterInsert()
+  logInsert() {
+    console.log('inserted Org', this.id);
+  }
+
+  // @BeforeInsert()
+  // checkAPI(){
+  //   console.log("checking the API", this.name, this.ein)
+
+  // }
```

Comment:
do we need all of the changes in this file? if so what are they doing?

Reply:
nope, they were just for my experimentation—forgot to remove! Thanks.